payment/controllers.go

```diff
@@ -286,6 +287,47 @@ func CreateOrder(service *Service) handlers.AppHandler {
 	})
 }
 
+// SetOrderTrialDaysInput - SetOrderTrialDays handler input
+type SetOrderTrialDaysInput struct {
+	TrialDays int64 `json:"trialDays" valid:"int"`
+}
+
+// SetOrderTrialDays is the handler for cancelling an order
+func SetOrderTrialDays(service *Service) handlers.AppHandler {
```

Comment:
I think you need a call to `ValidateOrderMerchantAndCaveats`

Reply:
addressed: https://github.com/brave-intl/bat-go/pull/945/commits/bc0dfc7a96b474900bad381e9553bed06c6491a1